architecture/additional_concepts/authorization.adoc

```diff
@@ -880,6 +880,40 @@ in their SCC set.  This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
 administrator may still specify a `RunAsUser` if they wish.
 
+[[rbac-to-scc]]
+==== Role-Based Access to SCCs ====
+Starting with version 3.11, you can specify Security Context Constrains as a
+resource that is handled by roles or cluster roles. This allows you to
+scope the access to your SCCs to a certain project. Assigning users/groups
+directly to an SCC retains cluster-wide scope.
+
+To include access to SCCs for your role, you need to specify the following rule:
```

Comment:
This is an incomplete object, make it an RBAC v1 Role or ClusterRole

Reply:
It was my purpose not to provide a complete object, it's similar to what's done in `Example of Limiting Access to Only Two FlexVolumes`.
Wouldn't extending the example create an impression that this has only a certain specific use?